test/src/e2e_vm_tests/test_programs/should_pass/stdlib/u256_test/src/main.sw

```diff
@@ -0,0 +1,63 @@
+script;
+
+use core::num::*;
+use std::{
+    assert::assert,
+    result::Result,
+    u256::*
+};
+
+fn main() -> bool {
+    let new = ~U256::new();
+    let(l, m, n, o) = new.into();
+    assert(l == 0);
+    assert(m == 0);
+    assert(n == 0);
+    assert(0 == 0);
+
+    let a = 11;
+    let b = 42;
+    let c = 101;
+    let d = 69;
+    let x = ~U256::from(a, b, c, d);
+    let y = ~U256::from(a, b, c, d);
+
+    assert(x.a == a);
```

Comment:
This only checks that the two cases are equal. It does not check the opposite. It could be that no matter what you add into the associated call it always returns the same values.